main.cpp

```diff
@@ -28,7 +28,19 @@ int main(){
       cout << "=" << num1*num2 << endl;
     }
     else if( input == "divide" ){
-      cout << "=" << num1/num2 << endl;
+      int remainder = num1%num2;
+
+      if( num2 == 0 ){
+        cout << "ERROR: Cannot divide by zero\n";
+      }
```

Comment:
Move this condition before the else if on line 30 or if possible merge the condition with line 33-35.